x-pack/plugins/dataset_quality/public/utils/default_timerange.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const DEFAULT_FROM_IN_MILLISECONDS = 24 * 3600000;
```

Comment:
**nit:** if this corresponds to a day, it might be more useful to call this `ONE_DAY`, it directly tells the amount of time it represents.

Reply:
Addressed in [da4cd0e](https://github.com/elastic/kibana/pull/172462/commits/da4cd0e73f307beaef1b6376930d6dafc64196bf).